### PR TITLE
README formatting fixes

### PR DESCRIPTION
**Pull request by bbrewington**
- /README.md: fix bold formatting
- /tools/bqman/README.md
  - alphabetic order
  - line length - generally tried to stick to the Google standard of 80 character limit

### CODE REVIEW

**Comment by agold-rh** [comment]
@bbrewington If you can fix the conflicts in README.md, I can get this merged.

**Comment by agold-rh** [comment]
@bbrewington If you can fix the conflicts, I'll get this merged asap.

**Comment by bbrewington** [comment]
@agold-rh resolved merge conflicts

once the CI checks finish, considering this good to go - lmk if there's anything else needing updated before merging (take a look at latest commit...made a couple more changes, still just in README)

I'd like to change "tf" to "tensorflow" throughout the repo where applicable - will tackle that in a different PR

**Review by agold-rh — APPROVED** [review]

**Pull request merged**
